packages/neon_framework/lib/src/blocs/apps.dart

```diff
@@ -50,7 +49,7 @@ abstract class AppsBloc implements InteractiveBloc {
   /// A subject emitting an event when the notifications page should be opened.
   BehaviorSubject<void> get openNotifications;
 
-  /// A collection of unsupported apps and their minimum required version.
+  /// A collection of apps and their version checks.
   BehaviorSubject<BuiltMap<String, VersionCheck>> get appVersionChecks;
```

Comment:
Is there a reason we expose all version checks?
I think we should have a map with only the unsupported apps.
This would move the logic from https://github.com/nextcloud/neon/pull/2027/files#diff-543b9d4d76180fb9419fc101e9ba86d855ec85948bcb43bfd6fc9685433785bbR57 into the bloc where it's supposed to be

Reply:
I did this because we will have to check in a few places which features are supported. Talk does it already really well by exposing the features through a list in the capabilities, but basically any other app needs to rely on the version. For example the version check will be important at runtime for https://github.com/nextcloud/neon/issues/2010 or endpoints only available through https://github.com/nextcloud/neon/issues/1492. Of course this will go away at some point once those features are supported by all versions we support, but that can take a while (minimum 1 year for server).

Reply:
That makes sense.
Please add the unupported apps as a getter to the app nevertheless.